lib/constants.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+ *
+ * This software is licensed to you under the Kinvey terms of service located at
+ * http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+ * software, you hereby accept such terms of service  (and any agreement referenced
+ * therein) and agree that you have read, understand and agree to be bound by such
+ * terms of service and are of legal age to agree to such terms with Kinvey.
+ *
+ * This software contains valuable confidential and proprietary information of
+ * KINVEY, INC and is subject to applicable licensing agreements.
+ * Unauthorized reproduction, transmission or distribution of this file and its
+ * contents is a violation of applicable laws.
+ */
+
+const constants = {
+  EnvironmentVariables: {
+    User: 'KINVEY_USER',
+    Password: 'KINVEY_PASSWORD'
```

Comment:
We have some other Kinvey command line utilities, and while I doubt the values used here would potentially pose a conflict, it would be a good idea to be slightly more verbose. E.g. `KINVEY_CLI_USER`, `KINVEY_CLI_PASSWORD`

Edit: Also, this format is consistent with the CLI session file name: https://github.com/Kinvey/kinvey-cli/blob/master/config/default.js#L28